kraken-std/src/kraken/std/python/tasks/pytest_task.py

```diff
@@ -7,10 +7,25 @@
 
 from kraken.common import flatten
 from kraken.core import Project, Property, TaskStatus
+from typing_extensions import Literal
 
 from .base_task import EnvironmentAwareDispatchTask
 
-# TODO (@NiklasRosenstein): Pytest coverage support
+CoverageFormat = Literal["xml", "html", "json", "lcov", "annotate", None]
+
+
+def get_coverage_path(format: CoverageFormat) -> str:
+    if format == "xml":
+        return ".xml"
+    elif format == "html":
+        return "_html"
```

Comment:
Why is it `_html` and not `.html`?

Reply:
Because for html and annotate pytest coverage creates a folder instead of a file.